### DISCUSSION TITLE
Popups labels in Android positioning wrong

### DISCUSSION THREAD

**Discussion by pjou8** [CommunityToolkit/Maui#1508] [Q&A]
I recently did an App with .NET MAUI that had no errors, but last day I opened it to try and update a new compilation, I found out that all the labels in the Popups appear way more on the bottom than intended, cutting the label, and if I try to apply a padding to rise it up, it appears more up but cut the same way.

This, is only happening on Android, while on iOS everything is still good as it was before the update.

The only thing I changed during that update was some updated to NuGet Packages (including the CommunityToolkit), and I checked other Maui projects I have with popups, and having the same packages with the same versions, I don't have this problem.

Has somebody found and solved this problem?

Thanks

**Comment by vhugogarcia** [comment]
Hello @pjou8 , could you please provide a sample repository? 🙂

**Comment by vhugogarcia** [comment]
@CliffAgius I think this discussion can be closed since there are already 2 issues that appears to be related to this discussion and we will follow the issue:
- https://github.com/CommunityToolkit/Maui/issues/1592
- https://github.com/CommunityToolkit/Maui/issues/1532

Let me know what you think :)

Thank you

**Comment by CliffAgius** [comment]
Closed and answered above. 😉
